static/scss/answers/collapsible-filters/collapsible-filters-templates.scss

```diff
@@ -45,6 +45,11 @@
 .AnswersVerticalGrid.CollapsibleFilters {
   @include CollapsibleFiltersOverrides;
 
+  .Answers-resultsWrapper {
+    width: 100%;
+    max-width: none;
```

Comment:
do we need to specify max-width: none here (which I think is the default value?)

Reply:
The max-width is set in theme.scss for all answers-container so I would need to override it here